test/models/host/subscription_facet_test.rb

```diff
@@ -90,6 +90,7 @@ def test_purpose_status_search
     end
 
     def test_update_purpose_status
+      Organization.any_instance.stubs(:simple_content_access?).returns(false)
```

Comment:
My preference would be to stub `host.organization` vs `Organization.any_instance` if possible here

Reply:
Updated, inspecting the attributes, I do see now it is better to use the host's organization compared to the other way.